### ISSUE TITLE
ImportError: cannot import name 'json' from 'itsdangerous'

### ISSUE THREAD

**Issue by MadBlade91** [pokealarm/pokealarm#810]
## Troubleshooting Checklist

- [x] My Scanner install is at the most recent version.
- [x] My PokeAlarm install is at the most recent version.
- [x] I've already checked the
 [FAQ](https://github.com/RocketMap/PokeAlarm/wiki/faq) and the
 [Wiki](https://github.com/RocketMap/PokeAlarm/wiki) already.

## Problem Description

After a git pull my docker container wont start anymore.
Heres the error message:
```
Traceback (most recent call last):

  File "./start_pokealarm.py", line 17, in <module>

    from flask import Flask, request, abort

  File "/usr/local/lib/python3.7/site-packages/flask/__init__.py", line 19, in <module>

    from . import json

  File "/usr/local/lib/python3.7/site-packages/flask/json/__init__.py", line 15, in <module>

    from itsdangerous import json as _json

ImportError: cannot import name 'json' from 'itsdangerous' (/usr/local/lib/python3.7/site-packages/itsdangerous/__init__.py)
```

## Environment
Operating System: ` Docker / Ubuntu Server 20.04 `
Python Version: ` 3.7 as the docker file says `
PokeAlarm Branch: ` master `
PokeAlarm Version: ` 26d115343a42077a1ec3a16fe407fa20bebfc8de `
Scanner used: `MAD/Master`

## PokeAlarm Setup:
Your config file: https://pastebin.com/hLz3PqaX

## Log Output
https://pastebin.com/XNYzFjb6

**Comment by MadBlade91** [comment]
itsdangerous==2.0.1 in requirement.txt fixed the issue.

dont forget to rebuild the container!
"docker-compose pull && docker-compose down && docker-compose build --no-cache && docker-compose up -d"

thanks @billyjbryant

**Comment by billyjbryant** [comment]
Fixed this with #811 pending PR merge
